docs/examples.txt

```diff
@@ -6,23 +6,89 @@ Examples
 Example Models
 --------------
 
-The Determined distribution contains several example experiments in the
-``examples/`` subdirectory. Each experiment consists of a single
-model definition and one or more experiment configurations.
+Determined includes several example machine learning models that have
+been ported to Determined's APIs. These examples can be found in the
+``examples/`` subdirectory of the `Determined GitHub repo
+<https://github.com/determined-ai/determined/tree/master/examples>`__;
+download links to each example can also be found below.
+
+Trial API
+~~~~~~~~~
+
+The examples below use the Trial API. Each example consists of a model
+definition, along with one or more experiment configuration files. To
+run one of these examples, download the appropriate ``.tgz`` file,
+extract it, ``cd`` into the directory, and use ``det experiment create``
+to create a new experiment, passing in the appropriate configuration
+file. For example, here is how to train the ``mnist_pytorch`` example
+with a fixed set of hyperparameters:
+
+.. code::
+
+    tar xzvf mnist_pytorch.tgz
+    cd mnist_pytorch
+    det experiment create const.yaml .
+
+.. list-table::
+  :header-rows: 1
+
+  * - Framework
+    - Dataset
+    - Filename
+  * - PyTorch
+    - MNIST
+    - :download:`mnist_pytorch.tgz </examples/mnist_pytorch.tgz>`
+  * - PyTorch
+    - CIFAR-10
+    - :download:`cifar10_cnn_pytorch.tgz </examples/cifar10_cnn_pytorch.tgz>`
+  * - TensorFlow (Estimator API)
+    - MNIST
+    - :download:`mnist_estimator.tgz </examples/mnist_estimator.tgz>`
+  * - TensorFlow (tf.keras)
+    - Fashion MNIST
+    - :download:`fashion_mnist_tf_keras.tgz </examples/fashion_mnist_tf_keras.tgz>`
+  * - TensorFlow (tf.keras)
+    - CIFAR-10
+    - :download:`cifar10_cnn_tf_keras.tgz </examples/cifar10_cnn_tf_keras.tgz>`
+  * - TensorFlow (tf.keras)
+    - Iris
+    - :download:`iris_tf_keras.tgz </examples/iris_tf_keras.tgz>`
+  * - Tensorpack
+    - MNIST
+    - :download:`mnist_tp.tgz </examples/mnist_tp.tgz>`
+
+Native API
```

Comment:
should we link to the quick-start guide above for the trial and then the tutorial for native: https://docs.determined.ai/latest/tutorials/native-tf-keras/index.html, here?